contracts/TokenFunctions.sol

```diff
@@ -182,17 +111,7 @@ contract TokenFunctions is Iupgradable {
      * @param _stakerAddress address of the Staker.
      */ 
     function getStakerAllLockedTokens(address _stakerAddress) external view returns (uint amount) {
```

Comment:
I couldn't find this function being used anywhere but the UI. If that's the case it'd be worth deprecating